fe/fe-core/src/main/java/com/starrocks/service/arrow/flight/sql/session/ArrowFlightSqlTokenManager.java

```diff
@@ -0,0 +1,95 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package com.starrocks.service.arrow.flight.sql.session;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import com.google.common.cache.RemovalNotification;
+import com.starrocks.encryption.EncryptionUtil;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.service.ExecuteEnv;
+import com.starrocks.sql.ast.UserIdentity;
+
+import java.math.BigInteger;
+import java.security.SecureRandom;
+import java.util.concurrent.TimeUnit;
+
+public class ArrowFlightSqlTokenManager implements AutoCloseable {
+
+    private final String arrowFlightSqlAseKey;
+
+    private LoadingCache<String, ArrowFlightSqlTokenInfo> tokenCache;
+
+    private final SecureRandom generator = new SecureRandom();
+
+    public ArrowFlightSqlTokenManager(String arrowFlightSqlAseKey) {
+        this.arrowFlightSqlAseKey = arrowFlightSqlAseKey;
+        this.tokenCache =
+                CacheBuilder.newBuilder()
+                        .maximumSize(1024)
```

Comment:
Put `1024` to Config.java to give a chance modify the max connection concurrency.